test/files/neg/t11962.scala

```diff
@@ -0,0 +1,8 @@
+// scalac: -Xlint -Xfatal-warnings
```

Comment:
```
// scalac: -Xlint -Werror -Ystop-after:refchecks
```
Prefer current -Werror. Stopping after refchecks is a nicety only.

Reply:
I followed [CONTRIBUTING.md](https://github.com/scala/scala/blob/2.13.x/CONTRIBUTING.md)  to use -Xfatal-warnings.
> To test that warnings are correctly emitted, use -Xfatal-warnings with a neg test and .check file. The usual way to create a .check file is partest --update-check.

Maybe CONTRIBUTING.md should be updated.

Reply:
Thanks for noticing that.